.circleci/config.yml

```diff
@@ -27,6 +27,7 @@ jobs:
             bundle install --path vendor/bundle
             sudo apt install lsof
             bundle exec spinach --tags simple-get
```

Comment:
I think you can do `bundle exec spinach --tags @simple-get, @head-to-simple-get `.

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/17/commits/6fd747773a9adb7e5d4ef5e1a21cfb63ff761dcb)